<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sketch</class>
 <widget class="QWidget" name="sketch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVM Dataset Generator </string>
  </property>
  <widget class="QPushButton" name="loadTif">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tif</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadGPS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GPS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadKml">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kml</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadPanos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Panos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geneTraj">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Traj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geneSat">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geneGrd">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Grd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geneCSV">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>340</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadOutputdir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output Dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="QlabelImg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>401</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="SatWidth">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sat width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Img interval</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ImgInterval">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>North Align</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="NorthAlign">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>140</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfigConfirm">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sat level</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SatLevel">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadAzi">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Azi</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPano">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>456</y>
     <width>821</width>
     <height>321</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>806</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Initial shift(deg.)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shiftDisplay">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>810</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>776</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>^</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>766</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt; positive angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>706</x>
     <y>766</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>negative angle &gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LULon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LULat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RBLon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RBLat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LU.Lon</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LU.Lat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RB.Lat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>67</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RB.Lon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Area">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Area</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
